packages/strapi-design-system/src/Link/Link.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { TableLabel } from '../Text';
+import { Box } from '../Box';
+
+const LinkWrapper = styled.a`
+  text-transform: uppercase;
+  text-decoration: none;
+  pointer-events: ${({ disabled }) => (disabled ? 'none' : undefined)};
+
+  svg path {
+    fill: ${({ disabled, theme }) => (disabled ? theme.colors.neutral600 : theme.colors.primary600)};
+  }
+
+  svg {
+    font-size: ${10 / 16}rem;
+  }
+
+  &:hover,
+  &:active {
+    color: ${({ theme }) => theme.colors.primary800};
+  }
+`;
+
+// TODO: make sure to use the link from the router library chosen
+export const Link = ({ href, to, children, disabled, leftIcon, rightIcon, ...props }) => {
+  const linkHref = disabled ? '#' : href || to;
+  const target = href ? '_blank' : undefined;
+  const rel = href ? 'noreferrer noopener' : undefined;
+
+  return (
+    <LinkWrapper target={target} rel={rel} href={linkHref} disabled={disabled} {...props}>
+      <TableLabel textColor={disabled ? 'neutral600' : 'primary600'} as="span">
```

Comment:
I am less and less comfortable with these Text component names, to be honest. TableLabel in links 🤔

Reply:
Yeah :( that's not very practical

Reply:
cc @maevalienard